### ISSUE TITLE
DDL region doesn't push changed event after continuous register

### ISSUE THREAD

**Issue by amyangfei** [pingcap/ticdc#452]
## Bug Report

Please answer these questions before submitting your issue. Thanks!

1. What did you do? If possible, provide a recipe for reproducing the error.

test workload from tipocket

2. What did you expect to see?

3. What did you see instead?

- region=6 doesn't receive event anymore.

```
[2020/04/10 11:42:37.285 +00:00] [WARN] [client.go:1029] ["region not receiving event from tikv for too long time"] [regionID=6] [span="{\"Start\":\"bURETEpvYkFk/2RJZHhMaXN0/wAAAAAAAAAA9wAAAAAAAABs\",\"End\":\"bURETEpvYkFk/2RJZHhMaXN0/wAAAAAAAAAA9wAAAAAAAABt\"}"] [duration=1m0s]
[2020/04/10 11:42:37.285 +00:00] [WARN] [client.go:1029] ["region not receiving event from tikv for too long time"] [regionID=6] [span="{\"Start\":\"bURETEpvYkxp/3N0AAAAAAAA+QAAAAAAAABs\",\"End\":\"bURETEpvYkxp/3N0AAAAAAAA+QAAAAAAAABt\"}"] [duration=1m0s]
[2020/04/10 11:42:37.285 +00:00] [WARN] [client.go:1029] ["region not receiving event from tikv for too long time"] [regionID=6] [span="{\"Start\":\"bURETEpvYkFk/2RJZHhMaXN0/wAAAAAAAAAA9wAAAAAAAABs\",\"End\":\"bURETEpvYkFk/2RJZHhMaXN0/wAAAAAAAAAA9wAAAAAAAABt\"}"] [duration=1m0s]
[2020/04/10 11:42:37.285 +00:00] [WARN] [client.go:1029] ["region not receiving event from tikv for too long time"] [regionID=6] [span="{\"Start\":\"bURETEpvYkxp/3N0AAAAAAAA+QAAAAAAAABs\",\"End\":\"bURETEpvYkxp/3N0AAAAAAAA+QAAAAAAAABt\"}"] [duration=1m0s]
```
- tikv logs: http://139.219.11.38:8000/13ZqHt/tipocket-tikv-20200410.tar.gz

```
[2020/04/10 11:41:37.279 +00:00] [INFO] [raft.rs:900] ["became leader at term 18"] [term=18] [raft_id=78] [region_id=6]

```
![WeChatWorkScreenshot_91b2faea-6091-44c6-bae3-dee17fac5914](https://user-images.githubusercontent.com/1527315/79035322-ece43380-7bef-11ea-8c9b-3b3a62ab2166.png)

4. Versions of the cluster

```
./cdc version
Release Version: v4.0.0-beta.2-60-g70a479c
Git Commit Hash: 70a479cb8107ab8b52b0c915e72c4def3ddda9c3
Git Branch: master
UTC Build Time: 2020-04-10 10:40:52
Go Version: go version go1.13.4 linux/amd64
```

```
./tikv-server -V
TiKV
Release Version:   4.1.0-alpha
Git Commit Hash:   c95f6ce36ad34ec74f9575c7c97fc22e1e21a9f6
Git Commit Branch: HEAD
UTC Build Time:    2020-04-08 10:31:21
Rust Version:      rustc 1.42.0-nightly (0de96d37f 2019-12-19)
Enable Features:   jemalloc portable sse protobuf-codec
Profile:           dist_release
```

**Comment by amyangfei** [comment]
This problem has been solved, but another problem is introduced.

**Comment by amyangfei** [comment]
```
./tikv-server -V
TiKV
Release Version:   4.1.0-alpha
Git Commit Hash:   9e37c9c63e4118776b98a4bbe9b734a8a405680c
Git Commit Branch: HEAD
UTC Build Time:    2020-04-22 08:54:00
Rust Version:      rustc 1.42.0-nightly (0de96d37f 2019-12-19)
Enable Features:   jemalloc portable sse protobuf-codec
Profile:           dist_release
```

The latest TiKV still has this bug.

**Comment by leoppro** [comment]
fixed in pingcap/tikv#7695
